modules/document_repository/README.md

```diff
@@ -0,0 +1,42 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides a centralized location for non-candidate documents that are relevant to the study (i.e. unfilled versions of instruments, publications, or manual of operations).
+
+## Intended Users
+
+ LORIS users can view, download, upload, and delete files, as well as edit information pertaining to those files.
+ This module is used primarily by study coordinators/administrators who share study protocols and manuals, as well as data entry staff who consult these documents.
+
+## Scope
+
+ Any LORIS user can edit and delete the all files in the Document Repository module. 
```

Comment:
Typo: "Any LORIS user can edit and delete ALL THE files in the Document Repository module"